Retry jobs (#4213)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/03-Oct-2023/native-retry-support-rn.md

---
title: "Enhancement: Native support for the dbt retry command"

> description: "October 2023: Rerun errored jobs from start and from the failure point"

Reviewer: This can be either or, right? 
```suggestion
description: "October 2023: Rerun errored jobs from start or from the failure point"
```

Author: not in this case, i don't think. it's more similar to this:

"foo supports x, y, and z"

Author: weird i had it on the page "Retry jobs"

sidebar_label: "Enhancement: Support for dbt retry"
tags: [Oct-2023, Scheduler]
date: 2023-10-06
sidebar_position: 10
---

> Previously in dbt Cloud, you could only rerun an errored job from start but now you can also rerun it from its of failure. 

Author: ```suggestion
Previously in dbt Cloud, you could only rerun an errored job from start but now you can also rerun it from its point of failure. 
```

> Previously in dbt Cloud, you could rerun an errored job from start but now you can rerun it from the job run's point of failure. 

Reviewer: Suggest calling out the two possible behaviors:
```suggestion
Previously in dbt Cloud, you could only rerun an errored job from start, but now you can either rerun a job from the start or from the job run's point of failure. 
```
